inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/consts/OpenApiConstants.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.inlong.audit.config;
+package org.apache.inlong.audit.consts;
```

Comment:
It seems this modification is not related to flink job parallelism, maybe we can do this in another pull request.

Reply:
This pull request depends on some constants in OpenApiConstants for the first time. However, I encountered some dependency problems when I tried to add the audit-service dependency to the maven-plugin module.  @vernedeng  suggested that this refactor may be helpful to resolve the dependency problem. Refactoring this in another pull request will cause compilation issues and lots of troubles.

Reply:
That makes sense, thanks for your explain.